common/src/main/scala/net/psforever/objects/avatar/PlayerControl.scala

```diff
@@ -95,69 +177,74 @@ class PlayerControl(player : Player) extends Actor
       super.CancelJammeredSound(obj)
     case _ => ;
   }
+
+  def RepairValue(item : Tool) : Int = if(player.ExoSuit != ExoSuitType.MAX) {
+    item.FireMode.Modifiers.Damage0
+  }
+  else {
+    item.FireMode.Modifiers.Damage3
+  }
```

Comment:
Isn't this always going to hit the first case of Damage0, unless I'm missing something where a player in a MAX can use a Tool to repair something?

Reply:
Damage and repair actions resolve in a target-centric context, i.e., the `player` here is the one being repaired, the one being managed by the `PlayerControl` agency, not the person doing the repairing (`user`).

Reply:
Ah yeah of course, that makes more sense. Even then, why the different behaviour is present isn't immediately obvious. MAXes have a different repair rate than other armour?

Reply:
The way we had figured these damage/repair values works like this:
`damage0` - vanilla infantry
`damage1` - armor
`damage2` - aircraft
`damage3` - mechanized infantry
`damage4` - unknown (used for amenities and some deployables)

Reply:
Realistically-speaking, however, the `bank` does not naturally benefit from this.  The `nano_dispener` does.  This part is included for customization potential.

Reply:
> The way we had figured these damage/repair values works like this:
> `damage0` - vanilla infantry
> `damage1` - armor
> `damage2` - aircraft
> `damage3` - mechanized infantry
> `damage4` - unknown (used for amenities and some deployables)

damage1 is base amentities and ground vehicles
damage4 is BFRs/battleframe robotics

Reply:
Changed.